recipe/hello.qmd

```diff
@@ -1,27 +1,11 @@
 ---
-title: "matplotlib demo"
+title: "hello world"
 format:
   html:
     code-fold: true
 jupyter: python3
 ---
 
-For a demonstration of a line plot on a polar axis, see @fig-polar.
-
 ```{python}
-#| label: fig-polar
-#| fig-cap: "A line plot on a polar axis"
-
-import numpy as np
-import matplotlib.pyplot as plt
-
-r = np.arange(0, 2, 0.01)
-theta = 2 * np.pi * r
-fig, ax = plt.subplots(
-  subplot_kw = {'projection': 'polar'}
-)
-ax.plot(theta, r)
-ax.set_rticks([0.5, 1, 1.5, 2])
-ax.grid(True)
-plt.show()
+print('hello world')
 ```
```

Comment:
Good call on simplifying this, especially since matplotlib was not listed as a test dependency. What do you think about adding an R example in here? The quoting is hard to do in this comment since it overlaps with github's markdown support, but  you should be able to just copy/paste what you have for hello world and replace `{python}` with `{r}`